projects/angular/src/emphasis/alert/alert.spec.ts

```diff
@@ -18,9 +18,9 @@ const CLOSE_ARIA_LABEL = 'Close Test Alert';
       [clrAlertType]="type"
       [clrAlertSizeSmall]="isSmall"
       [clrAlertClosable]="isClosable"
+      [clrAlertLightweight]="isLight"
```

Comment:
Maybe just let's keep the same naming:

`[clrAlertLightweight]="isLightweight"`

Reply:
Corrected in commit 2224898